### PR TITLE
Update tutorials with built-in service instances [ECR-4219]

### PR DESCRIPTION
**Pull request by slowli**
This PR updates cryptocurrency tutorials to reflect changes brought by exonum/exonum#1765.

------

See: https://jira.bf.local/browse/ECR-4219

### CODE REVIEW

File: src/get-started/create-service.md [hunk 1]
```diff
@@ -614,6 +614,19 @@ impl Service for CryptocurrencyService {
 }
 ```
 
+### Default Instantiation Params
```
Comment (popzxc) [inline src/get-started/create-service.md]:
```suggestion
### Default Instantiation Parameters
```

File: src/get-started/create-service.md [hunk 2]
```diff
@@ -626,192 +639,60 @@ The code we are going to write is logically separate from the service itself.
...
 while the demo blockchain is a specific example of its usage. For this reason,
-we will position the blockchain code as an [*example*][cargo-example] and
+we will position the blockchain code as an [example][cargo-example] and
 place it into [`examples/demo.rs`][demo.rs].
 
 ### Additional Dependencies
 
 Since services themselves do not require the Exonum node, in this example we
-want to create one, and interact with it as well. Thus, we have to add several
+want to create one, and interact with it as well. Thus, we have to add
```
Comment (popzxc) [inline src/get-started/create-service.md]: I think "several" was used to prevent "add additional" tautology.

Comment (slowli) [reply src/get-started/create-service.md]: Ah, I see. But now, there is only one dependency mentioned in the code snippet below - this was the reason for removing "several." I'll look for a way around this.

File: src/get-started/create-service.md (hunk 1 above)
Comment (slowli) [reply src/get-started/create-service.md]: IMO, "Params" is OK for a title; the table of contents is visually large as is for these articles 🙃

File: src/get-started/create-service.md [hunk 3]
```diff
@@ -626,192 +639,60 @@ The code we are going to write is logically separate from the service itself.
...
-we will position the blockchain code as an [*example*][cargo-example] and
+we will position the blockchain code as an [example][cargo-example] and
 place it into [`examples/demo.rs`][demo.rs].
 
 ### Additional Dependencies
 
 Since services themselves do not require the Exonum node, in this example we
-want to create one, and interact with it as well. Thus, we have to add several
-additional dependencies in our `Cargo.toml`:
+want to create one, and interact with it as well. Thus, we have to more
```
Comment (popzxc) [inline src/get-started/create-service.md]: Now "we have to more dependencies" sounds awkward :)

Comment (popzxc) [reply src/get-started/create-service.md]: I'd suggest to either use "we have to specify more dependencies" or "we have one more dependency".

**Pull request merged**